docs/fundamentals/architecture-overview.md

```diff
@@ -0,0 +1,26 @@
+# Architecture Overview
+
+{ref}`Executor <executor>` and {ref}`Flow <flow>` are the two fundamental concepts in Jina. 
+
+The figure below shows details on how `Flow` and `Executor` abstractions translate into concrete entities, providing all the 
```

Comment:
```suggestion
The figure below shows details on how the Flow and Executor abstractions translate into concrete entities, providing all the 
```

In this context we're talking about Flow and Executor as concepts, not code. So we can drop the backticks